src/test/java/jenkins/advancedqueue/PriorityConfigurationPlaceholderTaskHelperTest.java

```diff
@@ -0,0 +1,164 @@
+    private static PriorityConfigurationPlaceholderTaskHelper helper;
+    private static Queue.Task ownerTask;
+    private static ExecutorStepExecution.PlaceholderTask task;
+
+    @BeforeClass
+    public static void setUp() throws IOException {
+        project = j.createFreeStyleProject();
+        helper = new PriorityConfigurationPlaceholderTaskHelper();
+        task = mock(ExecutorStepExecution.PlaceholderTask.class);
+        ownerTask = mock(Queue.Task.class);
+    }
+
+    @BeforeClass
+    public static void createActionAndItem() {
+        action = new Action() {
+
+            @Override
+            public String getIconFileName() {
+                return "";
+            }
+
+            @Override
+            public String getDisplayName() {
+                return "";
+            }
+
+            @Override
+            public String getUrlName() {
+                return "";
+            }
+        };
+        List<Action> actions = new ArrayList<>();
+        actions.add(action);
+        item = new Queue.WaitingItem(Calendar.getInstance(), project, actions);
+    }
+
+    @BeforeClass
+    public static void createStrategy() {
+        strategy = new PriorityStrategy() {
+            @Override
+            public boolean isApplicable(Queue.Item item) {
+                return false;
+            }
+
+            @Override
+            public int getPriority(Queue.Item item) {
+                return 3;
+            }
+
+            @Override
+            public void numberPrioritiesUpdates(int oldNumberOfPriorities, int newNumberOfPriorities) {}
+
+            @Override
+            public Descriptor<PriorityStrategy> getDescriptor() {
+                return null;
+            }
+        };
+    }
+
+    @BeforeClass
+    public static void createCallback() {
+        callback = new PriorityConfigurationCallback() {
+            /**
+             * @param indent
+             * @param log
+             * @return
+             */
+            @Override
+            public DecisionLogger addDecisionLog(int indent, String log) {
+                return null;
+            }
+
+            /**
+             * @param priority
+             * @return
+             */
+            @Override
+            public PriorityConfigurationCallback setPrioritySelection(int priority) {
+                return null;
+            }
+
+            /**
+             * @param priority
+             * @param jobGroupId
+             * @param reason
+             * @return
+             */
+            @Override
+            public PriorityConfigurationCallback setPrioritySelection(
+                    int priority, int jobGroupId, PriorityStrategy reason) {
+                return null;
+            }
+
+            /**
+             * @param priority
+             * @param sortAsInQueueSince
+             * @param jobGroupId
+             * @param reason
+             * @return
+             */
```

Comment:
Javadoc that only restates the names of the arguments is a distraction (to me anyway).  Since there is nothing special about the implementation and this is in a test, let's not include a Javadoc comment.
```suggestion
```